modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -165,6 +168,56 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets, clipboar
     icon: 'table'
   });
 
+  const tableClassList = getTableClassList(editor);
+
+  if (tableClassList.length === 0) {
+    // eslint-disable-next-line no-console
+    console.error('Missing table class list');
```

Comment:
I'm not sure what this is attempting to do, but this is definitely the wrong thing to be doing here.

Reply:
This is as per the spec.

Reply:
I haven't seen an official spec for this, so it's likely only a draft version. This is not how we do things in TinyMCE for an optional setting, so where this is mentioned it needs correcteing.

Reply:
Very well. I'll remove the error-logging.